app/objects/c_operation.py

```diff
@@ -259,9 +261,19 @@ async def run(self, services):
             while not await self.is_closeable():
                 await asyncio.sleep(10)
             await self.close(services)
+
+            # Automatic event log output
+            file_svc = services.get('file_svc')
+            await self.write_event_logs_to_disk(file_svc, data_svc, output=True)
         except Exception as e:
             logging.error(e, exc_info=True)
 
+    async def write_event_logs_to_disk(self, file_svc, data_svc, output=False):
+        event_logs = await self.event_logs(file_svc, data_svc, output=output)
+        event_logs_dumps = json.dumps(event_logs)
+        r_dir = await file_svc.create_exfil_sub_directory('%s/event_logs' % self.get_config('reports_dir'))
```

Comment:
Can `r_dir` be renamed to something more descriptive? Is it supposed to mean `reports_dir`?

Reply:
Yes, thank you! I'll go ahead and change it